lib/editors.sql

```diff
@@ -36,5 +36,5 @@ SELECT
 FROM
   questions AS q
 WHERE
-  q.course_id = $course_id
+  q.course_id = $course_id -- should be left the same?
```

Comment:
Yep. This is for file editing, which will always be within the question's course.